frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx

```diff
@@ -110,7 +110,7 @@ const AttachDeploymentToOBCModal = withHandlePromise((props: AttachDeploymentToO
     e.preventDefault();
     handlePromise(k8sPatch(DeploymentModel, requestDeployment, getPatches()))
       .then((res) => {
-        history.push(resourceObjPath(res, referenceFor(res)));
+        history.push(`${resourceObjPath(res, referenceFor(res))}/environment`);
```

Comment:
Why this change ?

Reply:
This is where the user can see the actual changes made by the attachment. We are adding env variables.